app/serializers/v2/repository_serializer.rb

```diff
@@ -16,8 +16,10 @@ class RepositorySerializer < ApplicationSerializer
     end
 
     link :self do
```

Comment:
Just one suggestion: Consider making the urls part of the model (generally). That way a model 'owns' and 'knows' it's location on the web, which is one of the key-informations. It also removes the need to potentially build urls in multiple places.